src/Altinn.Platform/Altinn.Platform.Storage/Storage/appsettings.json

```diff
@@ -22,7 +22,10 @@
     "AccountName": "devstoreaccount1",
     "AccountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
     "StorageContainer": "servicedata",
-    "BlobEndPoint": "http://127.0.0.1:10000/devstoreaccount1"
+    "BlobEndPoint": "http://127.0.0.1:10000/devstoreaccount1",
+    "OrgKeyVaultURI": "https://{0}-{1}-keyvault.vault.azure.net/",
+    "OrgStorageAccount": "{0}altinnat{1}storage01",
+    "OrgSasDefinition": "{0}{1}sasdef01"
```

Comment:
As previously discussed these could be defined for each environment. {1} worries me as we use both 'Production' and 'PR' to identify production, and naming standard for azure resources might not mach Startup.EnvironmentName which it is being replaced by

Reply:
The values are environment agnostic and in this case I prefer to have it like that instead. I did replace Startup.EnvironementName with a new GeneralSettings.EnvironmentName and added values to ops project for storage.

Reply:
Changed my mind again. The other reason to use the values.yml file is to hide them from the public repo.